packages/blocks/src/api/parser.js

```diff
@@ -26,6 +26,28 @@ import { attr, html, text, query, node, children, prop } from './matchers';
 import { normalizeBlockType } from './utils';
 import { DEPRECATED_ENTRY_KEYS } from './constants';
 
+/**
+ * Options for parse.
+ *
+ * @property {Object<string,Function>} sources An object defining custom source
```

Comment:
I don't think this JSDoc matches. Might be a copy/paste thing?

Reply:
> I don't think this JSDoc matches. Might be a copy/paste thing?

Oof. Good call. Updated to something more reasonable in the rebased 272e660.

Reply:
👍